### DISCUSSION TITLE
Update a date string

### DISCUSSION THREAD

**Discussion by lmilbaum** [renovatebot/renovate#26462] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

github.com

### What is your question?

Can renovate update the following date/time string with the current date/time?
I'd hopped a custom manager would be of value in this case.

```
    createdAt: "2023-12-28 05:12:39"
```

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by secustor** [comment]
Currently not possible

**Reply by rarkins** [reply]
Loose sorting wouldn't work?

**Reply by secustor** [reply]
Could be and there is probably an API to get the current time which can be used with the custom datasource, tough this will constantly create and update PRs. 

Which is not what is asked here, but rather to update a timestamp if there is an upgrade.

**Reply by lmilbaum** [reply]
@secustor You are correct. Maybe a pre upgrade task could be used but that probably requires a custom image
